.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
This is not related to current project, it should be in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)